content/en/blog/2022-05-10-may-2022-security-announcement.md

```diff
@@ -0,0 +1,288 @@
+---
+author: 'dholbach & pjbgf'
+date: 2022-05-10 8:30:00+00:00
+title: May 2022 Security Announcement
+description: New Flux and Flagger releases bring you the latest and greatest just yet! KubeCon news, GitOps Days coming up and lots of new resources - potentially the most newly added videos in particular! Lots of great contributions from our community. 💖
```

Comment:
I was thinking we could use the TL;DR... 

```suggestion
description: The Flux Team has found three security vulnerabilities in Flux, Today we will go through them and talk about what this may mean to you. We strongly advise you to upgrade your clusters as soon as you can. :lock: 
```